app/src/main/AndroidManifest.xml

```diff
@@ -34,7 +42,7 @@
             android:stopWithTask="false"
             android:exported="false">
             <intent-filter>
-
+                <action android:name="com.google.firebase.INSTANCE_ID_EVENT" />
```

Comment:
I have a feeling we may not need this. Can you double check?

Reply:
removed